src/main/java/task/Test.java

```diff
@@ -0,0 +1,32 @@
+package task;
+
+import java.time.LocalDateTime;
+import parser.DateTimeExtractor;
+
+/**
+ * This extension of the Event class will allow the student to add Tests to
+ * his schedule.
+ *
+ * @author Varun
+ */
+public class Test extends Event {
+
+    public Test(String description, LocalDateTime toDate, LocalDateTime atDate) {
```

Comment:
If Test does not have any specialized behavior from Event, then what is purpose of having this Class?